server/handler.go

```diff
@@ -197,13 +210,43 @@ func getRecipes(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	//for local work
```

Comment:
This is a bad approach to leave commented code.  You can move it to function and add flag to config (local or prod)